docs/Compose.md

```diff
@@ -0,0 +1,181 @@
+# Running GUAC with Docker Compose
+
+This tutorial introduces the full GUAC ingestion pipeline and how to deploy it
+with the included Docker Compose configuration.
+
+## Demo GUAC Ingestion
+
+The GUAC demonstrations use the full GUAC GraphQL server, but use a demo-level
```

Comment:
i think let's skip this line since reading this without context, i got a bit confused which demo this is refering to ( i had assumed it was this demo, which is incorrect)

Reply:
I rewrote this paragraph, main point is to explain why someone would want to run compose, vs do what is in the demos.